wizeline/project-lab
# Feat-#265-adding the innovation tiers

Added the functionality of setting a innovation tier to the projects.
Also I created a new table _InnovationTiers_ and a new field on the Project table to relation both tables.

By default all the projects gets the _Tier 3 (Experiment)_ tier. Only the managers can modify this value.



#### How should this be manually tested?

1. Go to the manager section
2. Go to the Innovation Tier tab
3. Add a new innovation tier
4. Go to the details of one project
5. Change the innovation tier of the project
6. Save and check again

#### Screenshots


#### Questions
Please add any comments or suggestions regarding the proposed structure


#### Checklist
